extension/src/CSV/ImportXliffCSV.ts

```diff
@@ -21,25 +20,27 @@ export function importXliffCSV(
   );
 
   if (importSettings.updateTargetStateFromCsv) {
-    requiredHeaders.push("State");
+    requiredHeaders.push(CSVHeader.state);
   }
```

Comment:
is it even possible to push to a `const`?!

Reply:
```
The const declaration creates a read-only reference to a value. It does not mean the value it holds is immutable—just that the variable identifier cannot be reassigned. For instance, in the case where the content is an object, this means the object's contents (e.g., its properties) can be altered. 
``` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#description